conformance/tests/generics_variance.py

```diff
@@ -52,10 +52,7 @@ def dump_employee(e: E) -> E:
 B_co = TypeVar("B_co", covariant=True)
 
 
-# > Variance is only applicable to generic types; generic functions do not
-# > have this property. The latter should be defined using only type variables
-# > without covariant or contravariant keyword arguments.
-def bad_func(x: list[B_co]) -> B_co:  # Type checker error
+def func(x: list[B_co]) -> B_co:  # OK
```

Comment:
```suggestion
# > Variance has no meaning, and should therefore be ignored by type checkers, if
# > a type variable is bound to a generic function or type alias.
def func(x: list[B_co]) -> B_co:  # OK
```
Quoting the new wording in the spec.

Also, could you add some test cases demonstrating the desired behavior around aliases?

Reply:
Added the quote and reran tests.

I think we have these cases covered in the compliance test suite already. They're further down in the same file.